kernel/aster-nix/src/fs/file_table.rs

```diff
@@ -137,7 +150,10 @@ impl FileTable {
             let events = FdEvents::Close(fd);
             self.notify_fd_events(&events);
             entry.notify_fd_events(&events);
-            closed_files.push(entry.file);
+            closed_files.push(entry.file.clone());
+            if let Some(inode_file) = entry.file.downcast_ref::<InodeHandle>() {
+                inode_file.release_range_locks();
```

Comment:
For improved clarity, rename `entry` to `removed_entry` and do `inode_file` to `closed_inode_file`.